cpu/sam0_common/periph/uart.c

```diff
@@ -52,9 +52,6 @@ static inline SercomUsart *_uart(uart_t dev)
 }
 
 #ifdef CPU_FAM_SAML21
-
-static uint64_t _long_division(uint64_t n, uint64_t d);
-
 static uint8_t sercom_gclk_id[] =
```

Comment:
Just one more question : 
Do you think we can move this array in cpu/sam0_common/include/periph_cpu_common.h ? (or somewhere else)
I'm working on a refactored I2C driver and I'll need this array. (Maybe it can be useful in the SPI driver too...)

Reply:
Sure, that makes much sense, indeed.